### PR TITLE
lost without map kata done

### PR DESCRIPTION
**Pull request by mpsujek**
It was simple but I can train map functions by that.

### CODE REVIEW

File: js/lost_without_map.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const maps = require('./lost_without_map');
+
+
+describe('maps', () => {
+
+  test('test NaN', () => {
```
Comment (Gelio) [inline js/lost_without_map.test.js]: How does it test `NaN`?

File: js/lost_without_map.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const maps = require('./lost_without_map');
+
+
+describe('maps', () => {
```
Comment (Gelio) [inline js/lost_without_map.test.js]: Let's improve the title 😄

File: js/lost_without_map.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// https://www.codewars.com/kata/beginner-lost-without-a-map/train/javascript
+
+function maps(x){
+  return x.map(function(el){
```
Comment (Gelio) [inline js/lost_without_map.js]: The formatting is a bit off

File: js/new_solution.test-ignore.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// const [function] = require('./[filename]);
```
Comment (Gelio) [inline js/new_solution.test-ignore.js]: Do we need this file?

File: js/lost_without_map.test.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const maps = require('./lost_without_map');
+
+
+describe('lost without map tests', () => {
+
+  test('Test-1 for simple multiplying 3 elements array', () => {
```
Comment (Gelio) [inline js/lost_without_map.test.js]: The full test name does not read like a sentence

> lost without map tests Test-1 for simple multiplying 3 elements array

Please improve it :smile: You can use my other suggestions for test names

File: js/new_solution.test.js [hunk 6]
```diff
@@ -1,10 +0,0 @@
-// const [function] = require('./[filename]);
```
Comment (Gelio) [inline js/new_solution.test.js]: Did you actually mean to delete this file :smile: ?

Comment (mpsujek) [reply js/new_solution.test.js]: It was intentional. I will prepare script to create new files.

**Pull request merged**